lib/class-wp-rest-block-editor-settings-controller.php

```diff
@@ -101,163 +101,163 @@ public function get_item_schema() {
 				'__unstableEnableFullSiteEditingBlocks'  => array(
 					'description' => __( 'Enables Full site editing blocks', 'gutenberg' ),
 					'type'        => 'boolean',
-					'context'     => array( 'view' ),
+					'context'     => array( 'post-editor', 'site-editor', 'widget-editor' ),
```

Comment:
@youknowriad, see this context which is meant to allow the mobile app to pick settings for a given block editor type. As discussed in https://github.com/WordPress/gutenberg/pull/31027#issuecomment-824151592, we need to figure out what those contexts should be.

Reply:
@draganescu, should it be `widget-editor` or `widgets-editor`?

Reply:
> @draganescu, should it be `widget-editor` or `widgets-editor`?

@talldan, @noisysocks, @tellthemachines, or @kevin940726 - can you help?

Reply:
`widgets-editor` seems to be preferred everywhere. Does this match another value defined in the codebase? The name of the editor passed to `gutenberg_initialize_editor` is currently `widgets_editor` with an underscore.

There's also the customizer screen, which is `widgets_customizer`.

Reply:
`widgets-editor` it is then. I updated the code in https://github.com/WordPress/gutenberg/pull/29969/commits/3897d19f7e41a81f11b98e2337de81169dab8949. Thank you.